Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -406,3 +430,24 @@ public enum LocalReactionState: String {
     /// Deleting of the reaction failed and cannot be fulfilled
     case deletingFailed
 }
+
+/// The type describing message delivery status.
+public struct MessageDeliveryStatus: RawRepresentable, Hashable {
+    public let rawValue: String
+    
+    public init(rawValue: String) {
+        self.rawValue = rawValue
+    }
+    
+    /// The message delivery state for message that is being sent/edited/deleted.
+    public static let pending = Self(rawValue: "pending")
```

Comment:
We can write these a little be shorter:
```suggestion
    public static let pending: Self = "pending"
```

Reply:
Yeah, I thought about this. `MessageDeliveryStatus` is not representable by string literal and implementing this protocol would require adding another `init`. I think saving a couple of chars should not be the reason for adding more complexity to sources

Reply:
AH never mind, I thought this was StringRepresentable, never mind then! :)